pages/components/Todolist.tsx

```diff
@@ -21,7 +23,17 @@ const Todolist = () => {
             setTodos([...todos, newTask]);
             setNewTodo('');
         }
-    }
+    };
+
+    const handleKeyPress = (e: React.KeyboardEvent<HTMLInputElement>) => {
+        if (e.key === 'Enter') {
+            if (editIndex > -1) {
+                saveTodo();
+            } else {
+                addTodo();
+            }
+        }
+    };
```

Comment:
I mean to remove this function, we can achieve it through combination of form and button

Reply:
Done now sir